pkg/client/volume.go

```diff
@@ -301,6 +310,14 @@ func (r *DateraClient) CreateVolume(name string, volOpts *VolOpts, qos bool) (*V
 				Size:          int(volOpts.Size),
 				PlacementMode: volOpts.PlacementMode,
 				ReplicaCount:  int(volOpts.Replica),
+				PerformancePolicy: &dsdk.PerformancePolicy{
+					WriteIopsMax: int(volOpts.WriteIopsMax),
+					ReadIopsMax: int(volOpts.ReadIopsMax),
+					TotalIopsMax: int(volOpts.TotalIopsMax),
+					WriteBandwidthMax: int(volOpts.WriteBandwidthMax),
+					ReadBandwidthMax: int(volOpts.ReadBandwidthMax),
+					TotalBandwidthMax: int(volOpts.TotalBandwidthMax),
+				},
```

Comment:
This is ok, it should work, but it leaves dead code around.

I think you should:
- remove the boolean QoS parameter from the CreateVolume method in volume.go
- remove the code that uses that parameter below (volume.go:347, as that is dead code now)
- remove the qos parameter when calling CreateVolume

Reply:
The tricky part here is that - removing QoS alone is not sufficient. There is another condition about empty template in that if condition. I can remove that condition too and run another regression to check if anything breaks.

Reply:
It's
`	if qos && volOpts.Template == ""
`

So, if qos is false (which is always the case now), then the code at volume.go:347 is dead code.